automation/test.sh

```diff
@@ -151,6 +151,24 @@ collect_debug_logs() {
     done
 }
 
+build_images() {
+    # build all images with the basic repeat logic
+    # probably because load on the node, possible situation when the bazel
+    # fails to download artifacts, to avoid job fails because of it,
+    # we repeat the build images action
+    local tries=3
+    for i in $(seq 1 $tries); do
+        if make bazel-build-images; then
+            return 0
+        else
+            rc=$?
+            if [ $i -eq $tries ]; then
+                return $rc
+            fi
+        fi
```

Comment:
Will this work?
```
for i in $(seq 1 3); do
    make bazel-build-images && return
    rc=$?
done

return $rc
```

Reply:
we have `-e` so i am not sure it will work, i can try if you think otherwise